app/translations/en.json

```
@@ -993,6 +995,18 @@
                     "label": "Sort by name",
                     "description": "If set to true, then blueprints will be sorted by name."
                 }
+            },
+            "authenticationWarning": "No GitHub credentials provided! Widget may stop working after reaching unrestricted query limit (~50). Fix this by adding &github-username& and &github-password& entries to your secrets store (Secrets widget).",
+            "uploadingMessage": {
+                "uploading": "Uploading {{blueprintName}} blueprint",
```

Maybe:
```suggestion
                "uploading": "Uploading '{{blueprintName}}' blueprint...",
```
to be inline with other parts of the application? (see for example "widgets.common.deployments.deployModal.steps" in `en.json`)

---

Sure, seems reasonable 💪